salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -7,9 +7,12 @@ edx:
         - DeviceIndex: 0
           AssociatePublicIpAddress: True
           SubnetId: {{ subnetids[id_num % num_subnets] }}
-          SecurityGroupId: |
-            {{ securitygroupids.edxapp, securitygroupids.edx-worker, securitygroupids.default,
-            securitygroupids.salt-master, securitygroupids.consul-agent }}
+          SecurityGroupId:
+            - {{ securitygroupids.edxapp }}
+            - {{ securitygroupids.edx-worker }}
```

Comment:
This isn't necessary on the edxapp instance since it's exposing a port that's already present in the edxapp security group.